server/proxy/service.go

```diff
@@ -33,6 +33,7 @@ type Proxy struct {
 
 type Service interface {
 	Call(sessionID, actingUserID string, creq *apps.CallRequest) *apps.ProxyCallResponse
+	CleanUserCallContext(actingUserID string, cc *apps.Context) (*apps.Context, error)
```

Comment:
2/5 I don't like that the permission check is split between the proxy and the caller. That makes it hard to verify that permission are checked.

I'm advocating for moving `CleanUserCallContext` out of the proxy and letting the `caller´ deal with checking the permission.

Reply:
Moved the function to the `restapi` package